tests/test_tiling.py

```diff
@@ -104,8 +104,8 @@ def test_overlap(self):
             ]
         )
         p_overlap = 0.5
-        transformed = TileTransform(
-            source,
+        transformed = source.transform(
+            TileTransform,
```

Comment:
how about source.transform("tile_transform", ...)?

Reply:
For the similarity, how about using Tile instead of TileTransform? If so, we can do source.transform("tile", ...).

Reply:
Datumaro automatically converts its name during plugin registration: `TileTransform` -> `tile`.

https://github.com/openvinotoolkit/datumaro/blob/da67b52f496283182eafb85add9892adeba6ddcd/datumaro/components/cli_plugin.py#L30-L33

Thus, `source.transform("tile", ...)` and `source.transform(TileTransform, ...)` are equivalent. But because it's been common practice if we look at the other transforms, e.g. `CropCoveredSegments`, I'll rename it into `Tile`.

Reply:
I got it. thanks.